src/main/java/com/eerussianguy/blazemap/feature/atlas/AtlasExporter.java

```diff
@@ -0,0 +1,24 @@
+package com.eerussianguy.blazemap.feature.atlas;
+
+import com.eerussianguy.blazemap.engine.BlazeMapAsync;
+
+public class AtlasExporter {
+    private static AtlasTask task = null;
```

Comment:
Nice refactor, splitting it all out like this! Also, `AtlasTask` is defs a better name than the old `Task`. Good stuff here!